apps/studio/components/interfaces/Integrations/ProjectLinker.tsx

```diff
@@ -349,20 +351,24 @@ const ProjectLinker = ({
                         })}
                         {foreignProjects.length === 0 && (
                           <p className="text-xs text-foreground-lighter px-2 py-2">
-                            No GitHub repositories found
+                            No results found
```

Comment:
put in `<CommandEmpty_shadcn_>`

you can remove the `<p>` element and classes I think

Reply:
done!